server/engine/src/main/antlr4/org/eclipse/lsp/cobol/core/CobolPreprocessorLexer.g4

```diff
@@ -66,7 +63,7 @@ SKIP2 : S K I P '2' DOT_FS? -> channel(HIDDEN);
 SKIP3 : S K I P '3' DOT_FS? -> channel(HIDDEN);
 
 // symbols
-DOUBLEEQUALCHAR : '==';
+DOUBLEEQUALCHAR : '==' -> pushMode(REPLACEMODE);
```

Comment:
spaces in the partial-word enclosed within ==, seems to be preserved and looks important when replacing pic clause, like below
` REPLACING ==:oi x(01):== BY ==pic x(05)==`
 
 becasue `pic x ( 0 5 )` would break the parsing as its happening in current development